Dockerfile

```
@@ -7,6 +7,10 @@
 # Every target has a BIN_NAME argument that must be provided via --build-arg=BIN_NAME=<name>
 # when building.
 
+# go-discover builds the discover binary
+FROM golang:1.20.4-alpine as go-discover
```

The `golang:1.20.7-alpine` image is available and the latest release. We should probably use that one.

```suggestion
FROM golang:1.20.7-alpine as go-discover
```